v03_pipeline/lib/tasks/update_variant_annotations_table_with_new_samples_test.py

```diff
@@ -1436,8 +1465,14 @@ def test_sv_update_vat(
             ],
         )
 
-    def test_gcnv_update_vat(self, mock_update_rdc_task: None) -> None:
+    def test_gcnv_update_vat(
+        self,
+        mock_update_rdc_task,
+        mock_vat_get_datasets_to_update,
+        mock_vat_globals_from_dataset_configs,
+    ) -> None:
         mock_update_rdc_task.return_value = MockCompleteTask()
+        mock_vat_get_datasets_to_update.return_value = []
```

Comment:
I'm a little concerned by the need to mock the other dataset types here.  There's no reference data, so the `datasets` array should be empty already, making those tasks complete by default.  Is there another approach here?